js/src/personal/maliha/ModalMenu.tsx

```diff
@@ -1,26 +1,31 @@
-import { useState } from 'react'
 import { useDisclosure } from '@mantine/hooks'
 import { Modal, UnstyledButton, Stack } from '@mantine/core'
 import { Sandwich } from './Sandwich.tsx'
 import styles from './Maliha.module.css'
 
-const labels = [{ label: 'about' }, { label: 'projects' }, { label: 'gallery' }]
+/* Menu Component */
+const labels = [
+  { label: 'About' },
+  { label: 'Projects' },
+  { label: 'Gallery' },
+  { label: 'Notes' },
+  { label: 'Resume' },
+]
 
 export function Menu() {
   const [opened, { open, close }] = useDisclosure(false)
-  const [burgerOpened, setBurgerOpened] = useState(false)
 
   const items = labels.map((label) => (
-    <UnstyledButton className={styles.menuItems}>{label.label}</UnstyledButton>
+    <UnstyledButton
+      key={label.label}
+      className={styles.menuItems}
+    >
+      {label.label}
+    </UnstyledButton>
   ))
 
   const handleToggleMenu = () => {
-    if (opened) {
-      close()
-    } else {
-      open()
-    }
-    setBurgerOpened(!burgerOpened)
+    opened ? close() : open()
```

Comment:
useDisclosure actually has `toggle()` built in: https://mantine.dev/hooks/use-disclosure/